src/main/java/by/home/main/Main.java

```diff
@@ -0,0 +1,33 @@
+package by.home.main;
+
+import by.home.CircleObservable;
+import by.home.CircleStore;
+import by.home.IdGenerator;
+import by.home.entity.Circle;
+import by.home.entity.Point;
+import by.home.logic.Axis;
+import by.home.logic.Calculator;
+
+public class Main {
+    public static void main(String[] args) {
```

Comment:
according to task you should not write main, try to use tests. If you write for your own then do not commit it